### ISSUE TITLE
GraphQL response shows Google reCaptcha secretKey and siteKey env var names rather than the data itself

### ISSUE THREAD

**Issue by toddpadwick** [verbb/formie#666]
We have set up Formie reCaptcha in the CMS and we're attempting to set up reCaptcha on the client-side in our Nuxt headless setup.
We have confirmed that the env vars are correctly stored on the server. And the env vars are added to formie in the reCaptcha admin panel. however, GraphQL response seems to show the env vars themselves rather than the actual data assigned to them.

![unnamed](https://user-images.githubusercontent.com/36191462/144267805-6cc0cc69-2107-4bbe-b78c-d555c198e901.png)

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: GQL
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Comment by engram-design** [comment]
This is sort of expected behaviour, because each integration has different settings, it'd be tricky to set an Interface for every provider, to control how environment variables are parsed. In a "normal" scenario, they're parsed only when they're used.

But I've also needed to clean up the settings for an integration, which contained a whole lot of unnecessary data.

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.27",
  "...": "..."
}
```

Then run `composer update`.
